src/utils/editor/source-documents.js

```diff
@@ -4,8 +4,10 @@ import { getMode } from "../source";
 
 import type { Source } from "debugger-html";
 import { isWasm, getWasmLineNumberFormatter, renderWasmText } from "../wasm";
+import { SourceEditorUtils } from "devtools-source-editor";
 
 let sourceDocs = {};
+let wasmLineNumberFormatters: WeakMap<Object, string> = new WeakMap();
```

Comment:
this is a nice solution, but i think we might want it to be a bit more generic.

For instance, could it be an editorMap?